pkg/apis/batch/types.go

```diff
@@ -167,6 +168,7 @@ type JobStatus struct {
 	// Represents time when the job was completed. It is not guaranteed to
 	// be set in happens-before order across separate operations.
 	// It is represented in RFC3339 form and is in UTC.
+	// The completion time is only set when the job finishes successfully
```

Comment:
Missing period. (above change, too)

Please make these changes in the public, versioned API, or they won't show up in the documentation.

Reply:
Done. Are the additional comments sufficient are does more work need to be done?